var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -50,6 +50,10 @@ class Mfem(Package):
     # If this quick verification procedure fails, additional discussion
     # will be required to verify the new version.
 
+    version('3.3',
+            'b17bd452593aada93dc0fee748fcfbbf4f04ce3e7d77fdd0341cc9103bcacd0b',
+            url='http://goo.gl/Vrpsns', extension='.tgz')
```

Comment:
Can you change this to `.tar.gz`, or change the old ones to `.tgz`? I'm not sure it makes a difference. They will be extracted the same way. I think the only difference is how we name the archive.